src/serial/msp.ts

```diff
@@ -210,14 +212,12 @@ export class MSPPassthrough {
 
   private async enable() {
     const version = await this.msp.send(MSPCmd.MSP_API_VERSION);
-    if (version.payload[1] < 1 || version.payload[2] < 42) {
+
+    const mspVersion = version.payload.map((n) => "" + n).join(".");
+    Log.debug("msp", "api version", mspVersion);
+    if (!semverSatisfies(mspVersion, MSP_MIN_VERSION)) {
       throw new Error("unsupported msp version");
```

Comment:
Let also add mspVersion to the error, similar to the other errors below.

Reply:
done